boards/arm/96b_wistrio/96b_wistrio.dts

```diff
@@ -61,7 +62,7 @@
 	status = "okay";
 	cs-gpios = <&gpiob 0 0>;
 
-	sx1276@0 {
+	lora_radio: sx1276@0 {
```

Comment:
Let's use `lora` as the node name. I've submitted a PR to add it in DT spec.

https://github.com/devicetree-org/devicetree-specification/pull/32